### PR TITLE
added more icons

### PR DESCRIPTION
**Pull request by ChopperTheKing**
image

### CODE REVIEW

**Review by k-code00 — APPROVED** [review]
flags look good. i can help you on the rest if needed

**Review by mattmaloof — APPROVED** [review]
Use constraints and tableView.

**Review by mattmaloof — COMMENTED** [review]
REMOVE DS STORE IT SHOULD NEVER BE Uploaded in git

File: DeBlockWallet/Assets.xcassets/img_group.imageset/img_group.png
Comment (gisele-prog) [inline DeBlockWallet/Assets.xcassets/img_group.imageset/img_group.png:1]: looks nice!

**Review by gisele-prog — APPROVED** [review]

**Pull request merged**